### PR TITLE
Adding OVHcloud

### PR DESCRIPTION
**Pull request by mhurtrel**
Adding CNCF Certfied Managed Kubernetes Service at OVHcloud

### CODE REVIEW

**Comment by mhurtrel** [comment]
I signed it

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-19111--kubernetes-io-master-staging.netlify.com/docs/setup/

File: content/en/docs/setup/_index.md [hunk 1]
```diff
@@ -100,6 +100,7 @@ The following production environment solutions table lists the providers and the
...
 | [OpenShift](https://www.openshift.com)          |[OpenShift Dedicated](https://www.openshift.com/products/dedicated/) and [OpenShift Online](https://www.openshift.com/products/online/)              |        | [OpenShift Container Platform](https://www.openshift.com/products/container-platform/)            |      |  [OpenShift Container Platform](https://www.openshift.com/products/container-platform/)        |[OpenShift Container Platform](https://www.openshift.com/products/container-platform/)
 | [Oracle Cloud Infrastructure Container Engine for Kubernetes (OKE)](https://docs.cloud.oracle.com/iaas/Content/ContEng/Concepts/contengoverview.htm)          | &#x2714;             |   &#x2714;     |             |      |          |
+ [OVHcloud](https://www.ovhcloud.com/en/public-cloud/)         | [Managed Kubernetes Service](https://www.ovhcloud.com/en/public-cloud/kubernetes/)              | [OVHcloud Public Cloud](https://www.ovhcloud.com/en/public-cloud/)        |             |     | &#x2714;          |
```
Comment (sftim) [inline content/en/docs/setup/_index.md]: This seems to be missing a leading `|`. Also, the name Managed Kubernetes Service seems very generic; it it in fact “OVHcloud Managed Kubernetes Service”?

In the future it's likely that this entry will be removed, along with others, following policy changes under discussion in https://github.com/kubernetes/enhancements/pull/1327

Comment (mhurtrel) [reply content/en/docs/setup/_index.md]: Indeed, was missing the initiatl pipe, thanks for the catch.
Now corrected. We went indeed for a descriptive name, as most of the industry.

**Comment by mhurtrel** [comment]
@kbhawkey any chance you add back lgtm tag? thanks :)
/assign kbhawkey

**Comment by kbhawkey** [comment]
Hello @mhurtrel .
There have been updates the KEP (regarding inclusion of third party links).
The relevant PR to follow is:
https://github.com/kubernetes/website/pull/19486

**Comment by jimangel** [comment]
/hold

pending #19486

**Pull request closed without merging**